lib/usage.js

```diff
@@ -122,9 +134,10 @@ module.exports = function (yargs, y18n) {
         return acc
       }, {})
     )
+    var theWrap = getWrap()
```

Comment:
On second thought, looks like it would add `getWrap()` to 3 other places in the code, which is a big ugly, so gonna suggest we just leave it as-is. Cost of calling `getWrap()` multiple times should be minimal.